Destination BigQuery + Snowflake : Disable type and dedupe step controlled by flag in Connector spec (#31686)
airbytehq/airbyte: docs/release_notes/upgrading_to_destinations_v2.md

We may make further changes to raw tables in the future, as these tables are intended to be a staging ground for Airbyte to optimize the performance of your syncs. We cannot guarantee the same level of stability as for final tables in your destination schema, nor will features like error handling be implemented in the raw tables.

> As a user previously not running Normalization, Upgrading to Destinations V2 will increase the compute costs in your destination data warehouse. This is because Destinations V2 will now be performing the operations to generate a final typed table (Check connector specific settings to disable generating Typed tables).

Reviewer: Nit:
```suggestion
As a user previously not running Normalization, Upgrading to Destinations V2 will increase the compute costs in your destination data warehouse. This is because Destinations V2 will now be performing the operations to generate a final typed table. Some destinations may provide an option to disable this - check your connectors's settings.
```
